pvlib/shading.py

```diff
@@ -342,3 +342,61 @@ def projected_solar_zenith_angle(solar_zenith, solar_azimuth,
     # Eq. (5); angle between sun's beam and surface
     theta_T = np.degrees(np.arctan2(sx_prime, sz_prime))
     return theta_T
+
+
+def linear_shade_loss(shaded_fraction, diffuse_fraction):
+    r"""
+    Fraction of power lost to linear shade loss applicable to monolithic thin
+    film modules like First Solar CdTe, where the shadow is perpendicular to
+    cell scribe lines.
+
+    .. versionadded:: 0.10.5
+
+    Parameters
+    ----------
+    shaded_fraction : numeric
+        The fraction of the collector width shaded by an adjacent row. A
+        value of 1 is completely shaded and zero is no shade.
+    diffuse_fraction : numeric
+        The ratio of diffuse plane of array (poa) irradiance to global poa.
+        A value of 1 is completely diffuse and zero is no diffuse.
+
+    Returns
+    -------
+    linear_shade_loss : numeric
+        The fraction of power lost due to linear shading. A value of 1 is all
+        power lost and zero is no loss.
+
+    Notes
+    -----
+    The reasoning behind this loss model is described in sections 4.2 and 4.3
+    of [1]_, page 8 of the manual.
+
+    The loss is calculated as:
+
+    .. math::
+
+        \text{loss} = \text{shaded\_fraction}\cdot(1-\text{diffuse\_fraction})
```

Comment:
```suggestion
        \text{loss} = \text{shaded_fraction}\cdot(1-\text{diffuse_fraction})
```
I don't think you need the backslashes in \text environment, at least they currently are rendered in the documentation.
![image](https://github.com/pvlib/pvlib-python/assets/39184289/a0e436d8-3011-45ba-9dcb-39373fd9b2bc)